src/Converter/Feature2Texture.js

```diff
@@ -74,7 +76,7 @@ const coord = new Coordinates('EPSG:4326', 0, 0, 0);
 function drawFeature(ctx, feature, extent, style, invCtxScale) {
     const extentDim = extent.planarDimensions();
     const scaleRadius = extentDim.x / ctx.canvas.width;
-    const globals = {
+    context.globals = {
```

Comment:
It can indeed be moved before calling drawFeature, but becasue of globals.zoom = extent.zoom (that was already set here before) it can't be moved where context is initialized.
In a futur PR we can change the way globals works. With a co,ntext.setZoom(extent.zoom) and maybe initialized the fill, stroke and point atinitialization.

Reply:
ah sorry I didn't see the last line that was hidden by github diff :grin: